index.html

```diff
@@ -1346,26 +1338,25 @@
       </h2>
       <pre class="idl">
         dictionary PaymentDetailsModifier {
-          required sequence&lt;DOMString&gt; supportedMethods;
+          required DOMString supportedMethods;
           PaymentItem total;
           sequence&lt;PaymentItem&gt; additionalDisplayItems;
           object data;
         };
       </pre>
       <p>
         The <a>PaymentDetailsModifier</a> dictionary provides details that
-        modify the <a>PaymentDetailsBase</a> based on <a>payment method
-        identifier</a>. It contains the following fields:
+        modify the <a>PaymentDetailsBase</a> based on a <a>payment method
+        identifier</a>. It contains the following members:
       </p>
       <dl>
         <dt>
           <dfn>supportedMethods</dfn>
```

Comment:
> Should this be renamed to supportedMethod, now that it only takes a single payment method identifier?

This already threw me off when just looking at the spec. The plural name with a string type is confusing… I was wondering how we're serializing a list into a string (which would be unusual given we could use an array)… Was an implementer not willing to change it?